### PR TITLE
Add a new folder for new installation with a script to resetting of the openmrs database to the default,ART Summary - Prior ART - if selected then all PEP, Hep-B, PMTCT and earlier ARV are required,Add Marital Status Living together. on registration and Merging Marital Status Single and Never Married.

### PR DESCRIPTION
**Pull request by slubwama**

### CODE REVIEW

File: tools/src/main/resources/create-sql-scripts.sh [hunk 1]
```diff
@@ -2,21 +2,27 @@
 
 # Turns off foreign key cheks to speed up script execution
 cat database/openmrs-platform.sql > upgrade/new-install.sql
+cat database/openmrs-platform.sql > new_install/new-install.sql
```
Comment (slubwama) [inline tools/src/main/resources/create-sql-scripts.sh]: openmrs-platform.sql has its own header. We cant have double headers

File: tools/src/main/resources/create-sql-scripts.sh [hunk 2]
```diff
@@ -2,21 +2,27 @@
...
 cat database/enable_birt_report_function.sql >> upgrade/new-install.sql
+cat database/enable_birt_report_function.sql >> new_install/new-install.sql
 
 # cleans up the data integrity module which will be re-installed
 cat database/cleanup_data_integrity_module.sql >> upgrade/new-install.sql
+cat database/cleanup_data_integrity_module.sql >> new_install/new-install.sql
 
 # Turns on foreign key checks
 cat database/mysql_script_footer.sql >> upgrade/new-install.sql
+cat database/mysql_script_footer.sql >> new_install/new-install.sql
```
Comment (slubwama) [inline tools/src/main/resources/create-sql-scripts.sh]: I think its just fine

File: omod/src/main/webapp/fragments/field/maritalStatus.gsp [hunk 3]
```diff
@@ -6,8 +6,8 @@
     <select name="${config.formFieldName}">
```
Comment (slubwama) [inline omod/src/main/webapp/fragments/field/maritalStatus.gsp]: @ssmusoke  Is this codding standard to other HMIS forms other than the ART card??

Comment (slubwama) [inline omod/src/main/webapp/fragments/field/maritalStatus.gsp]: @ssmusoke  Is this coding standard to other HMIS forms other than the ART card??

Comment (ssmusoke) [inline omod/src/main/webapp/fragments/field/maritalStatus.gsp]: @slubwama This is the style that I am advocating for to ease the burden on the data entry staff who follow the codes

File: tools/src/main/resources/create-sql-scripts.sh [hunk 4]
```diff
@@ -18,6 +18,9 @@ cat database/cleanup_data_integrity_module.sql >> upgrade/new-install.sql
 # Turns on foreign key checks
 cat database/mysql_script_footer.sql >> upgrade/new-install.sql
 
+#Copying new-install to new_install directory
+cp upgrade/new-install.sql new_install/
```
Comment (ssmusoke) [inline tools/src/main/resources/create-sql-scripts.sh]: @slubwama Include the file name being copied to remove ambiguity

File: omod/src/main/webapp/fragments/field/maritalStatus.gsp [hunk 5]
```diff
@@ -5,13 +5,12 @@
...
-        <option value ="dcd70b18-30ab-102d-86b0-7a5022ba4115">Married</option>
-        <option value ="dcd743c3-30ab-102d-86b0-7a5022ba4115">Divorced</option>
-        <option value ="dcd74997-30ab-102d-86b0-7a5022ba4115">Separated</option>
-        <option value ="dcd77876-30ab-102d-86b0-7a5022ba4115">Widowed</option>
+        <option value ="dce180a8-30ab-102d-86b0-7a5022ba4115">0 - Child</option>
+        <option value ="dc96cf5e-30ab-102d-86b0-7a5022ba4115">1 - Never Married/Single</option>
+        <option value ="dc96e101-30ab-102d-86b0-7a5022ba4115">2 - Living Together</option>
+        <option value ="dcd70b18-30ab-102d-86b0-7a5022ba4115">3 - Married</option>
+        <option value ="dcd743c3-30ab-102d-86b0-7a5022ba4115">4 - Divorced/Separated</option>
+        <option value ="dcd77876-30ab-102d-86b0-7a5022ba4115">6 - Widowed</option>
```
Comment (ssmusoke) [inline omod/src/main/webapp/fragments/field/maritalStatus.gsp]: @slubwama This option should be 5 not 6

**Comment by ssmusoke** [comment]
@slubwama The build is still failing, you need to merge the changes from my PR which fixes the referenceapplicaiton version issue

**Pull request merged**